paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/FormFragmentArguments.kt

```diff
@@ -0,0 +1,46 @@
+package com.stripe.android.paymentsheet.paymentdatacollection
+
+import android.os.Parcelable
+import com.stripe.android.paymentsheet.specifications.IdentifierSpec
+import kotlinx.parcelize.Parcelize
+
+@Parcelize
+data class FormFragmentArguments(
+    val supportedPaymentMethodName: String,
+    val saveForFutureUseInitialVisibility: Boolean,
+    val saveForFutureUseInitialValue: Boolean,
+    val merchantName: String,
+    val billingDetails: BillingDetails? = null,
+) : Parcelable
+
+@Parcelize
+data class Address(
```

Comment:
Is this being redefined just because of the dependency direction? Will it change to be the `PaymentSheet.Address`?

Reply:
yeah